action.yml

```diff
@@ -19,7 +19,7 @@ inputs:
     required: false
     default: balena-cloud.com
   versionbot:
-    description: Does the repository use VersionBot ?
+    description: (Deprecated) Does the repository use VersionBot?
```

Comment:
@maggie0002 adding these brackets could break the action.

Reply:
It seems very unlikely in yaml like this, but its GitHub actions, so I wouldn't bet the bank on it.

`description: Deprecated - Does the repository use VersionBot?`?